ega-charts/cega/values.yaml

```diff
@@ -1,23 +1,26 @@
 # Default values for localega.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-base:
-  image: egarchive/lega-base
-  imageTag: release.v0.1.0
-  imagePullPolicy: Always
-
 persistence: 
   enabled: true
 
 secrets:
-  cega_users_pass: 
+  cega_users_pass: ""
+
+tls: true
```

Comment:
missing indentation ?

Reply:
No it's a global var for all of fake cega.

Reply:
ok.